lib/engines/content_object_store/app/controllers/content_object_store/content_block/editions/publications_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ContentObjectStore::ContentBlock::Editions::PublicationsController < ContentObjectStore::BaseController
```

Comment:
I have a little bit of a reservation here as we have said we'll try to keep to Whitehall naming as much as poss - is the `create` here actually the equivalent of `publish` in Whitehalls `Edition_Workflow_Controller`? I could see if we started to have draft/superseded states etc that a Workflow Controller would be useful.
And is this now only used by the 'New' user journey?

Reply:
Ah, fair enough. I was trying to keep it RESTful, but I think it makes sense to keep it aligned